app/src/main/java/org/mozilla/fenix/searchdialog/SearchDialogFragment.kt

```diff
@@ -324,6 +337,66 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
             })
     }
 
+    override fun onRequestPermissionsResult(
+        requestCode: Int,
+        permissions: Array<String>,
+        grantResults: IntArray
```

Comment:
Maybe needs to be renamed, it isn't intuitive to me right away what this variable means (I assumed the result of whether the permissions have been granted?) Maybe something like permissionsResults or permissionsGrantedResults (kind of long)

Reply:
unfortunately it's an `override` function, so I can't rename 😢 however, I can add some comments & kdocs

Reply:
No worries then :D